about-pulp-3.md

```diff
@@ -0,0 +1,46 @@
+---
+title: About Pulp 3
+sidebar: home_sidebar
+permalink: /about-pulp-3/
+toc: false
+---
+
+Pulp 3 builds on the achievements of Pulp 2, with a renewed focus on repository management. Pulp 3 is more than an update of Pulp 2. Pulp 3 is the result of years of feedback from Pulp 2 production scenarios and serves to address key concerns that could not be resolved within the existing Pulp 2 architecture.
+
+Pulp 3 reduces the challenges and dependencies that users experienced with Pulp 2. Built on Django, and using open standards such as OpenAPI, Pulp 3 has increased reliability and flexibility while reducing the code base. Plug-in writing has become easier thanks to the improved Plug-in API. Performance speeds, data integrity, and safety features, such as roll back to specific versions, have guided the development of Pulp 3.
+
+Pulp 3 and Pulp 2 have several conceptual differences. For more information about conceptual and terminology differences between Pulp 3 and Pulp 2, see [Renamed concepts](https://docs.pulpproject.org/from-pulp-2.html#renamed-concepts).
+
+## Pulp 3 vs Pulp 2 At A Glance
+
+Pulp 3 has the following advantages and features when compared to Pulp 2:
+
+* Improved performance and data integrity. Pulp 3 moves from MongoDB to PostgreSQL.
+* Faster performance. On testing, synchronizing in Pulp 3 ranges from two to eight times faster than Pulp 2.
+* Versioned repositories feature. New ability to rollback safely to earlier versions.
+* Disk storage and speed solutions. Select from [multiple synchronization](https://docs.pulpproject.org/workflows/on-demand-downloading.html) options for all content types to suit specific environmental requirements.
```

Comment:
Any chance we could make cloud storage solutions a separate bullet point and link to https://docs.pulpproject.org/installation/storage.html?

Reply:
Hi @daviddavis - of course. Could you help me with the point? I'm looking for the main change between Pulp 2 vs Pulp 3 on this point and the main advantage for users :)

Reply:
How about: 

```
* More storage options. Store content in the cloud with Pulp 3's [support for storage services like S3 and Azure Blob](https://docs.pulpproject.org/installation/storage.html)
```

Reply:
Sounds good. Will add that!

Reply:
Added. Thanks